lib/foreman_build_history/facet.rb

```diff
@@ -1,7 +1,2 @@
 Facets.register(BuildHistoryFacet) do
```

Comment:
We can get rid of the `do..end` block entirely.
At least until we want to add a per-host history tab.